### DISCUSSION TITLE
Kovan Deprecated and in Goerli, DAI/ETH price feed is missing in chainlink docs

### DISCUSSION THREAD

**Discussion by ManiBharathiSS** [smartcontractkit/full-blockchain-solidity-course-py#1690] [Q&A] [answered]
Hey guys!  
              I am in lesson 10. In the video, @PatrickAlphaC  is using kovan DAI/ETH price feed. But since its is deprecated, i was trying to use goerli testnet. But in chain link docs DAI/ETH price feed is missing for goerli. Any ideas to come around this problem.

Thanks in advance

**Comment by PatrickAlphaC** [comment]
Skip the testnet part of the course, thanks!

**Comment by bensacc** [comment, marked as answer]
@ManiBharathiSS (@PatrickAlphaC I encountered the same thing and noticed Goerli does have contracts for DAI/USD and ETH/USD.  So, it's a bit more effort but you could implement both of these and then just compute the cross-currency rate and use that.  I'm having some other issues with Goerli at the moment, but I tested it on a mainnet-fork and got the following result:

Pdb) dai_eth_price
0.000714220165096707
(Pdb) dai_usd_price
1.00039645e-10
(Pdb) eth_usd_price
1.38852736923e-07
(Pdb) dai_usd_price / eth_usd_price
0.0007204729789048121
(Pdb) dai_usd_price * (1 / eth_usd_price) <-- technically, you would want DAI/USD * USD/ETH, but these are mathematically equivalent)
0.0007204729789048121

a bit more work, and they're not quite exactly the same (I mean, it is a crypto testnet)  but as an alternative to just not being able to do it on Goerli, it's an option...

**Reply by PatrickAlphaC** [reply]
Awesome, thanks for posting this!
